Lib/test/test_call.py

```diff
@@ -26,6 +26,18 @@ def fn(**kw):
         self.assertIsInstance(res, dict)
         self.assertEqual(list(res.items()), expected)
 
+    def test_frames_are_popped_after_failed_calls(self):
```

Comment:
Not that it matters much, but maybe you want to add the CPython specific decorator

Reply:
I thought about that, but I figure that this should still pass on any CPython implementation. If you somehow crash here, it's a bug, right? 😉

Reply:
Yeah, we have been approaching these kind of tests in different ways so it doesn't matter :)